src/main/cms/cms.c

```diff
@@ -992,7 +996,11 @@ void cmsUpdate(uint32_t currentTimeUs)
 
     if (!cmsInMenu) {
         // Detect menu invocation
+#ifdef USE_USB_CDC_HID
+        if (getBatteryCellCount() && IS_MID(THROTTLE) && IS_LO(YAW) && IS_HI(PITCH) && !ARMING_FLAG(ARMED)) {
```

Comment:
i still think this should move to line 980 similar to USE_RC_DEVICE.
then we dont need this 2 different if´s here